Engine/src/Eri/Platform/Linux/PlatformLinux.cpp

```diff
@@ -0,0 +1,224 @@
+#include "Eri/Platform/Linux/PlatformLinux.h"
+
+#include <time.h>
+#include <X11/Xlib.h>
+
+#include "Eri/Utils/Events/Mouse/EventMouse.h"
+#include "Eri/Platform/Linux/LinuxMouse.h"
+#include "Eri/Platform/Linux/LinuxKeys.h"
+
+// https://www.tronche.com/gui/x/xlib-tutorial/2nd-program-anatomy.html
+
+namespace ERI
+{
+void PlatformLinux::registerUtils(EriUtils *const utils)
+{
+  _log = utils->getLogger();
+  _events = utils->getEventSystem();
+}
+
+bool PlatformLinux::Startup()
+{
+  if (_log == nullptr || _events == nullptr)
+  {
+    return false;
+  }
+
+  _windowX = 0;
+  _windowY = 0;
+  _windowWidth = 0;
+  _windowHeight = 0;
+
+  _log->LogInfo("Created Platform System type Linux");
+  return true;
+}
+
+bool PlatformLinux::Shutdown()
+{
+  _log = nullptr;
+  _events = nullptr;
+
+  XDestroyWindow(_display, _window);
```

Comment:
Another thing to note. On my system, I am unable to kill my window. I'm not sure if it's a wayland thing, or a window manager thing, or a 'I didn't do X11 correctly' thing. ... but either fix or issue track